app/views/form/_deal_index_form.erb

```diff
@@ -39,8 +39,15 @@
                         </td>
                     <% else %>
                         <td>
-                          <%= link_to 'Push Deal', push_deal_path(deal),
-                             class: "btn btn-xs btn-success"%>
+                          <% if @payment.add_on2 %>
```

Comment:
I thought only if they paid for deal statistics will they know how many users they are pushing their deal to?